packaging/msi/Ice.wixproj

```diff
@@ -5,16 +5,49 @@
         Suppress warning from the VC++ redistributable merge modules
         see: https://wixtoolset.org/docs/v3/howtos/redistributables_and_install_checks/install_vcredist/#a-note-about-ice-warnings
       -->
-      <SuppressIces>ICE03;ICE82</SuppressIces>
+      <SuppressIces>ICE03;ICE30;ICE82</SuppressIces>
+
+      <StagingDir Condition="'$(StagingDir)' ==  ''">$(MSBuildThisFileDirectory)staging</StagingDir>
   </PropertyGroup>
 
   <ItemGroup>
-      <BindPath Include="$(MBuildThisFileDirectory)..\..\" BindName="SOURCE_ROOT" />
+      <BindPath Include="$(MSBuildThisFileDirectory)..\..\" BindName="SOURCE_ROOT" />
+      <BindPath Include="$(StagingDir)" BindName="STAGING_DIR" />
       <BindPath Include="$(VCInstallDir)\Redist\MSVC\v143\MergeModules\" BindName="MERGE_MODULES" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="WixToolset.Util.wixext" Version="5.0.2" />
     <PackageReference Include="WixToolset.UI.wixext" Version="5.0.2" />
   </ItemGroup>
+
+  <!-- Files from the NuGet build -->
+  <ItemGroup>
+    <Slices Include="$(PackageDir)slice\**\*.ice" />
```

Comment:
The Slice files from NuGet is the subset we want to distribute. Avoid having to set excludes here.